### PR TITLE
Fix - 417 -  Adding Settings Link Inside Drop Down Menu

### LINKED ISSUE #417: Addings Settings Link Inside Drop Down Menu
**I'm submitting a ...**  (check one with "x")
- [x] feature request

**Current behavior:**

There is no link to Settings.html in the drop-down menu in chrome bot.

screenshot from 2018-10-05 16-05-27

**Would you like to work on the issue?**

Please let us know if you can work on it or the issue should be assigned to someone else.
```
Yes

### PR DESCRIPTION
**Pull request by sk9331657**
Fixes #417 #### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.

- [x] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:

Added Settings in drop down

screenshot from 2018-10-05 16-09-16

#### Changes proposed in this pull request:

-Added Settings Html inside drop down

### CODE REVIEW

**Comment by sk9331657** [comment]
@Akshat-Jain please review.

**Comment by sk9331657** [comment]
@hongquan @mariobehling  @gabru-md  Please review it.

**Comment by fragm3** [comment]
@sk9331657 PR: https://github.com/fossasia/susi_chromebot/pull/388 solves this.

**Comment by sk9331657** [comment]
> @sk9331657 PR: #388 solves this.

PR #388 Needs improvement, till then this can be merged.

**Comment by fragm3** [comment]
@sk9331657 I don't think it works this way. The reviewers are yet to review it, making issue of already solved things is not justified.

**Comment by sk9331657** [comment]
@sk9331657 Actually, I didn't see your pull completely, I thought this issue is not solved.  Next time please clearly mention the updates in Pr title.

**Comment by ms10398** [comment]
@fragm3 you have closed your PR so should I review this PR? or you will reopen the PR

**Comment by fragm3** [comment]
@ms10398 I had created a new PR yesterday as the closed one had conflicts. Link to new PR: https://github.com/fossasia/susi_chromebot/pull/420. I think this should be closed

**Comment by sk9331657** [comment]
@ms10398  This can be merged bcoz it just a small issue and as @fragm3.   PR's are facing merge conflicts.

**Comment by rupav** [comment]
Duplicate of #388 and #420. So can be closed.

**Comment by sk9331657** [comment]
@rupav  Ok .

**Comment by sk9331657** [comment]
@Akshat-Jain  Please merge this also.

**Comment by sk9331657** [comment]
@gabru-md @Akshat-Jain @ms10398  Please  merge this.

**Comment by sk9331657** [comment]
@championballer  @ms10398  Please merge this one.

**Comment by sk9331657** [comment]
@akshatnitd  Please review.

**Comment by sk9331657** [comment]
@mariobehling  Please review this .

**Comment by sk9331657** [comment]
@akshatnitd   #415

File: src/popup.html [hunk 1]
```diff
@@ -9,7 +9,8 @@
     <link rel="stylesheet" href="https://unpkg.com/leaflet@1.3.1/dist/leaflet.css"
    integrity="sha512-Rksm5RenBEKSKFjgI3a41vrjkw4EVPlJ3+OiI65vTjIdo9brlAacEuKOiQ5OFh7cOI1bkDwLqdLw3Zg0cRJAAQ=="
    crossorigin=""/>
-    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+   <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css" integrity="sha384-mzrmE5qonljUremFsqc01SB46JvROS7bZs3IO2EmfFsd15uHvIt+Y8vEf7N7fWAU" crossorigin="anonymous">
```
Comment (akshatnitd) [inline src/popup.html]: Add an entire CSS file for a single icon is not desirable. Please use svg instead.

Comment (gabru-md) [reply src/popup.html]: please make the changes and ask for a review again once you're done with it.
thank you. cheers! :smiley:

**Comment by sk9331657** [comment]
@akshatnitd  Now it ready to be merged.

**Pull request merged**